src/parsers/manifestjson.js

```diff
@@ -121,8 +121,22 @@ export default class ManifestJSONParser extends JSONParser {
         }
       }
     }
+
     if (support.firefox_android) {
-      const versionAddedAndroid = support.firefox_android.version_added;
+      let versionAddedAndroid;
+      if (Array.isArray(support.firefox_android)) {
```

Comment:
I doubt that a gap in the supported versions would ever end into the `firefox` compat data, but given that "theoretically" both `firefox` and `firefox_android` should supports the same allowed formats, maybe it is reasonable to do the same thing for both just in case. 

@willdurand what is your opinion?

Reply:
Yeah I guess we could do the same for `firefox` but as you mentioned above, we can't be sure that this will ever be correct so I don't really know. I initially chose the highest min version because that avoids thinking about gaps, etc. even if it is restricting.

Reply:
@willdurand that's a good point, maybe we should just:
- add a comment nearby that `if (support.firefox) {...}` block that does briefly mention why we are not doing the same thing we do for the `firefox_android` one (a inline comment nearby the `firefox_android` checks may also be good, just a small one that briefly describe why we are doing it and why the last range is good enough for that purpose).
- add one small test that validates the format of the webextension api metadata that we get from the `@mdn/browser-compat-data` dependency, a test that will make us sure that we will notice if update to this dependency has changes to the format of the data that we don't expect
 
how that sounds?

Reply:
> add one small test that validates the format of the webextension api metadata that we get from the `@mdn/browser-compat-data` dependency, a test that will make us sure that we will notice if update to this dependency has changes to the format of the data that we don't expect

yep, that works for me.

Reply:
sounds good to me